Remotion/ObjectBinding/Web.Test/TestForm.aspx

```diff
@@ -14,49 +14,60 @@
  % You should have received a copy of the GNU Lesser General Public License
  % along with re-motion; if not, see http://www.gnu.org/licenses.
 --%>
-
 <%@ Page Language="c#" CodeBehind="TestForm.aspx.cs" AutoEventWireup="false" Inherits="OBWTest.TestForm" %>
 
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
   <title>Test Form</title>
   <remotion:HtmlHeadContents ID="HtmlHeadContents" runat="server" />
-  <script type="text/javascript">
+</head>
+<body>
+  <form id="form1" runat="server">
+  <asp:ScriptManager ID="ScriptManager" runat="server" EnablePartialRendering="true"
+    AsyncPostBackTimeout="3600" />
+  <input id="StringField" type="text" />
+  <input id="IntField" type="text" />
+  <input type="button" value="ASP.NET Ajax" onclick="DoAspNetAjaxCall()" />
+  <input type="button" value="jQuery Ajax" onclick="DoJQueryAjaxCall()" />
+  <div id="Result">
+  </div>
+  </form>
+<script type="text/javascript">
     function DoAspNetAjaxCall()
```

Comment:
why move this into the body? I think splitting this would be the choice that better follows the existing design. Keep the declarations up top and move the invocations to the end of the page.